keycloakClient/keycloakClient.go

```diff
@@ -1313,3 +1320,39 @@ func (c *Client) InitiateLogin(realmName string, loginURLEncoded InitiatePKCELog
 	}
 	return resp.RawResponse, nil
 }
+
+// Access Control Policy Endpoints
```

Comment:
if ease of navigation / separation of concerns is the objective of this comment, consider moving these methods into a `client_applications.go` file